### PR TITLE
Clarify `--add-prefetch` build flag

### PR DESCRIPTION
**Pull request by FredKSchott**
Clarified this build flag since a lot of people (myself included) weren't sure what it was doing. Also confirmed it's still working as intended.

/cc @justinfagnani

### CODE REVIEW

File: src/build/build.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ const buildDirectory = 'build/';
 
 export interface BuildOptions {
   swPrecacheConfig?: string;
-  insertDependencyLinks?: boolean;
+  prefetchDepedencies?: boolean;
```
Comment (justinfagnani) [inline src/build/build.ts]: "insert" seemed meaningful here, in that the effect of this flag is to insert prefetch link tags, the effect isn't to do prefetching.

File: src/commands/build.ts [hunk 2]
```diff
@@ -45,11 +45,12 @@ export class BuildCommand implements Command {
           'used for service worker generation.'
     },
     {
-      name: 'insert-dependency-links',
+      name: 'add-prefetch',
       type: Boolean,
-      description: 'Flatten dependency tree downloads by inserting ' +
-          'additional `<link rel="prefetch">` tags into ' +
-          'entrypoint and `<link rel="import">` tags into fragments and shell.'
+      description: 'Add dependency tree prefetching by inserting ' +
```
Comment (justinfagnani) [inline src/commands/build.ts]: I'd remove "tree", we haven't used the term "dependency tree" before. Maybe:

"Enable dependency prefetching by inserting..."

File: src/commands/build.ts [hunk 3]
```diff
@@ -45,11 +45,12 @@ export class BuildCommand implements Command {
           'used for service worker generation.'
     },
     {
-      name: 'insert-dependency-links',
+      name: 'add-prefetch',
```
Comment (justinfagnani) [inline src/commands/build.ts]: similar to the above comment,  I think "insert" is meaningful here. Maybe `insert-prefetch-links`?

Comment (FredKSchott) [inline src/commands/build.ts]: I prefer the shorter `add-prefetch` when it comes to command line args. And `insert` always felt like it was inserting something new into my application while `add` feels like it's adding a new feature/ability to my application. So I prefer as-is, but happy to make the change if you feel strongly.

File: src/build/build.ts (hunk 1 above)
Comment (FredKSchott) [inline src/build/build.ts]: Good point, changed naming to `addPrefetchLinks` to be in line with the cli flag.

File: src/commands/build.ts (hunk 3 above)
Comment (justinfagnani) [inline src/commands/build.ts]: I'm not really concerned about long flag names for things that'll hopefully be in a config file, if they're more descriptive. I think `insert-prefetch-links` is the most clear name.

Comment (FredKSchott) [inline src/commands/build.ts]: fine by me, I'll make the change

**Comment by FredKSchott** [comment]
@justinfagnani name changed, PTAL

**Pull request merged**